### DISCUSSION TITLE
Why/how is this free?

### DISCUSSION THREAD

**Discussion by will-stone** [will-stone/browserosaurus#481] [Q&A] [answered]
Why/how is this free?

**Comment by will-stone** [comment, marked as answer]
Browserosaurus has by far been the most enjoyable project I have ever created and this comes from the fact that I use it every day. Being a fan of open source software, I thought I would share it with the world. If you love Browserosaurus as much as I do, please consider sponsoring its future maintenance. Thank you.

**Comment by natis** [comment]
Just wanted to drop a note and say that I appreciate the work you did to create this. I did go with the v15 release as I'm someone that was using Choose Wisely for as long as I could. I prefer the ability to just have the browsers I use in the palette using the hide functionality in v15. The move to the bottom of the list functionality of v16 just felt clunky personally. But again, thank you for creating and sharing this with us.

**Comment by alentodorov** [comment]
Thank you Will for building this. I was afraid at first that it would eat a lor of CPU/RAM because of Electron but I'm delighted by how well it works on a iMac 2012. Thank you for building this.

**Reply by will-stone** [reply]
No worries. Glad to hear it works well on your machine 🙂
